geth/whisper/request_historic_messages.go

```diff
@@ -70,118 +74,143 @@ func RequestHistoricMessagesHandler(nodeManager common.NodeManager) rpc.Handler
 }
 
 type historicMessagesRequest struct {
-	Peer     []byte              //mailbox peer
-	TimeLow  uint32              //resend messages from
-	TimeUp   uint32              //resend messages to
-	Topic    whisperv5.TopicType //resend messages by topic
-	SymkeyID string              //Mailbox symmetric key id
-	PoW      float64             //whisper proof of work
-}
+	// MailServer enode address.
+	Peer []byte
 
-func parseArgs(args ...interface{}) (historicMessagesRequest, error) {
-	var (
-		r = historicMessagesRequest{
-			TimeLow: uint32(time.Now().Add(-24 * time.Hour).Unix()),
-			TimeUp:  uint32(time.Now().Unix()),
-		}
-	)
+	// TimeLow is a lower bound of time range (optional).
+	// Default is 24 hours back from now.
+	TimeLow uint32
+
+	// TimeUp is a upper bound of time range (optional).
+	// Default is now.
+	TimeUp uint32
 
+	// Topic is a regular Whisper topic.
+	Topic whisperv5.TopicType
+
+	// SymKeyID is an ID of a symmetric key used to authenticate to MailServer.
+	// It's derived from MailServer password.
+	SymKeyID string
+
+	// PoW is Whisper Proof of Work value.
+	PoW float64
+}
+
+func parseArgs(args []interface{}) (r historicMessagesRequest, err error) { //nolint: gocyclo
```

Comment:
Why do we need to change `...interface{}` into `[]interface{}`?

Reply:
The call would look like this `paseArgs(args...)` and here we would do `parseArgs(args ...interface{})`. I don't see a reason for passing params in this way as simple slice works the same.